client/entrypoints/express-checkout/index.js

```diff
@@ -114,31 +128,56 @@ jQuery( function ( $ ) {
 
 			const shippingRates = getShippingRates();
```

Comment:
Maybe this is not needed anymore? Since now we are using `option.shippingRates`. Although below I can see `shippingRates` being passed 🤔 .

Reply:
This is the part that actually adds it to the `options` object, and passes that object to `createExpressCheckoutElement()` below.